libraries/chain/controller.cpp

```diff
@@ -5503,8 +5515,8 @@ bool controller::should_terminate() const {
    return my->should_terminate();
 }
 
-bool controller::should_terminate(block_num_type head_block_num) const {
-   return my->should_terminate(head_block_num);
+bool controller::should_terminate(block_num_type head_block_num, block_num_type lib) const {
```

Comment:
why expose this API in the controller interface.

Reply:
Removed